scripts/build_and_release_image.sh

```diff
@@ -1,37 +1,41 @@
 #!/bin/bash
 set -e
 
-get_current_pcm_version() {
-  npm --prefix ./frontend pkg get version | tr -d '"'
+USAGE="$(basename "$0") [-h] [--tag YYYY.MM (defaults to current year and month] [--revision REVISION]"
```

Comment:
nit-pick: close the round bracket for my sanity :D

Reply:
Done in next rev